acceptance-tests/src/test/java/tech/pegasys/ethsigner/tests/general/RequestsAcceptanceTest.java

```diff
@@ -0,0 +1,106 @@
+package tech.pegasys.ethsigner.tests.general;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static tech.pegasys.ethsigner.tests.dsl.Accounts.RICH_BENEFACTOR;
+import static tech.pegasys.ethsigner.tests.dsl.Gas.GAS_PRICE;
+import static tech.pegasys.ethsigner.tests.dsl.Gas.INTRINSIC_GAS;
+
+import tech.pegasys.ethsigner.tests.AcceptanceTestBase;
+
+import java.io.IOException;
+import java.math.BigInteger;
+
+import org.assertj.core.data.Offset;
+import org.junit.Test;
+import org.web3j.protocol.core.DefaultBlockParameterName;
+import org.web3j.protocol.core.Request;
+import org.web3j.protocol.core.methods.request.Transaction;
+import org.web3j.protocol.core.methods.response.EthAccounts;
+import org.web3j.protocol.core.methods.response.EthGetBalance;
+import org.web3j.protocol.core.methods.response.EthGetTransactionCount;
+import org.web3j.utils.Convert;
+
+public class RequestsAcceptanceTest extends AcceptanceTestBase {
+
+  static final String RECIPIENT = "0x1b00ba00ca00bb00aa00bc00be00ac00ca00da00";
+  static final String TRANFER_AMOUNT_ETHER = "10";
+  static final BigInteger TRANSFER_AMOUNT_WEI =
+      Convert.toWei(TRANFER_AMOUNT_ETHER, Convert.Unit.ETHER).toBigIntegerExact();
+
+  @Test
+  public void getAccount() throws Exception {
+
+    EthAccounts ethAccountsRequest = ethSigner().jsonRpc().ethAccounts().send();
+
+    assertThat(ethAccountsRequest.getAccounts().size() == 1).isTrue();
+    String account = ethAccountsRequest.getAccounts().get(0);
+    assertThat(ethNode().accounts().balance(account)).isNotNull();
+  }
+
+  @Test
+  public void getTransactionCount() throws Exception {
+    Request<?, EthGetTransactionCount> ethGetTransactionCountRequest =
+        ethSigner()
+            .jsonRpc()
+            .ethGetTransactionCount(RICH_BENEFACTOR.address(), DefaultBlockParameterName.LATEST);
+
+    transaction();
+
+    EthGetTransactionCount ethGetTransactionCount = ethGetTransactionCountRequest.send();
+    BigInteger startTransactionCount = ethGetTransactionCount.getTransactionCount();
+
+    transaction();
+
+    ethGetTransactionCount = ethGetTransactionCountRequest.send();
+    BigInteger endTransactionCount = ethGetTransactionCount.getTransactionCount();
+
+    assertThat(startTransactionCount.add(BigInteger.ONE).equals(endTransactionCount)).isTrue();
+  }
+
+  @Test
+  public void getBalance() throws Exception {
+    Request<?, EthGetBalance> ethGetBalanceRequest =
+        ethSigner().jsonRpc().ethGetBalance(RECIPIENT, DefaultBlockParameterName.LATEST);
+
+    transaction();
+
+    EthGetBalance ethGetBalance = ethGetBalanceRequest.send();
+    BigInteger startBalance = ethGetBalance.getBalance();
+
+    transaction();
+
+    ethGetBalance = ethGetBalanceRequest.send();
+    BigInteger endBalance = ethGetBalance.getBalance();
+
+    assertThat(endBalance)
+        .isCloseTo(startBalance.add(TRANSFER_AMOUNT_WEI), Offset.offset(BigInteger.ONE));
+  }
+
+  private void transaction() throws IOException {
```

Comment:
A rule across Pantheon (setting the standard for PegaSys products) is no private methods in Acceptance Tests

Reply:
OK, I can just make it class scope ...